.github/workflows/lint.yaml

```diff
@@ -34,10 +34,13 @@ jobs:
       - name: install nono
         run: cargo install cargo-nono
       # `mc-sgx-quote-verify` should be `no_std`, but it isn't yet.  See #34
+      # `mc-sgx-core-types` should be `no_std`, but cargo nono doesn't parse
+      # correctly, see https://github.com/hobofan/cargo-nono/pull/64 for a
+      # suggested improvement
       - run: |
           cargo metadata --no-deps --format-version=1 | \
             jq -r '.packages[].name' | \
-            grep -v -e mc-sgx-core-build -e mc-sgx-dcap-quoteverify-sys -e mc-sgx-dcap-ql-sys -e mc-sgx-quote-verify -e mc-sgx-urts -e mc-sgx-capable-sys | \
+            grep -v -e mc-sgx-core-types -e mc-sgx-core-build -e mc-sgx-dcap-quoteverify-sys -e mc-sgx-dcap-ql-sys -e mc-sgx-quote-verify -e mc-sgx-urts -e mc-sgx-capable-sys | \
```

Comment:
This should pass with `rand_core`, no?

Reply:
unfortunatly not
```
➜  sgx git:(feature/key_request) cargo nono check --package mc-sgx-core-types
mc-sgx-core-types: SUCCESS
bitflags: SUCCESS
mc-sgx-core-sys-types: SUCCESS
rand_core: FAILURE
  - Source code contains an explicit `use std::` statement.
   --> /home/nick/.cargo/registry/src/github.com-1ecc6299db9ec823/rand_core-0.6.3/src/error.rs
```

Looking at the file:
```rust
#[cfg(feature = "std")] use std::boxed::Box;   
```
it looks like cargo nono is using some kind of regex parsing or similar, at least for the explicit std check, even manually moving the use to the next line results in the failure.